tests/cli/NewFwkLib/NewFwkLib.csproj.in

```diff
@@ -84,7 +84,8 @@
       <HintPath>${NUNIT}\bin\framework\nunit.framework.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <ProjectReference Include="$(CMAKE_BINARY_DIR)\clicache\src\Apache.Geode.vcxproj">
+    <ProjectReference Include="$(CMAKE_BINARY_DIR)\clicache\src\Apache.Geode.vcxproj"> <Project>{03A77323-4CE8-3C6C-87CE-530E8B5559DA}</Project>
```

Comment:
The project GUID should not be in here as each time the project is generated the GUID will change.

Reply:
done.